src/generic/toolbkg.cpp

```diff
@@ -358,26 +376,50 @@ bool wxToolbook::EnablePage(wxWindow *page, bool enable)
     return EnablePage(pageIndex, enable);
 }
 
+int wxToolbook::PageToToolId(size_t page) const
+{
+    wxASSERT_MSG(page < GetPageCount(), "Invalid page number");
```

Comment:
It seems that should be `wxCHECK_MSG()` as otherwise we're still going to crash dereferencing null page pointer in the next line.